client/src/containers/AlbumList.js

```diff
@@ -0,0 +1,70 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { Link } from 'react-router-dom';
+import styled from 'styled-components';
+
+import { endpoint } from '../helpers/config';
+
+import { fetchAlbums } from '../actions';
+
+const AlbumListWrapper = styled.div`
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: flex-start;
+`;
+
+const AlbumListItem = styled.div`
+  margin: 10px 10px 40px 10px;
+  width: 12.4%;
+  height: 170px;
+  text-align: center;
+  font-weight: 800;
+`;
+
+const AlbumCover = styled.div`
+  padding-bottom: 140px;
+  background: url(${props => props.src}) no-repeat;
+  background-size: cover;
+`;
+
+class AlbumList extends Component {
+  componentDidMount = () => {
+    const { fetchAlbums } = this.props;
+
+    fetchAlbums();
+  };
+
+  renderAlbums = () => {
+    const { albums } = this.props;
+
+    return (
+      albums.length > 0 &&
+      albums.map(album => {
+        return (
+          <AlbumListItem key={album.id}>
+            <Link to={`/album/${album.id}`}>
+              <AlbumCover src={endpoint + album.artwork_path} />
```

Comment:
`endpoint + album.artwork_path` is OK, but for consistency sake, you should be using template strings. (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals)